airbyte-integrations/bases/standard-destination-test/src/main/resources/format_reference.txt

```diff
@@ -0,0 +1,13 @@
+VALID FORMATS:
```

Comment:
what is the purpose to have this file?

Reply:
Sherif asked to add this (https://github.com/airbytehq/airbyte/issues/5712#issue-981524364): 

> It is probably helpful to list them in a text file in the directory linked above, just for reference e.g: just write out all the formats in a formats_reference.txt file.